include/dlaf/eigensolver/reduction_to_band/mc.h

```diff
@@ -69,17 +69,17 @@ using PanelT = matrix::Panel<panel_type, T, Device::CPU>;
 template <Coord panel_type, class T>
 using ConstPanelT = PanelT<panel_type, const T>;
 
-template <class T>
-T computeReflector(const comm::IndexT_MPI rank_v0, comm::Communicator& communicator,
-                   const std::vector<TileT<T>>& panel, SizeType j) {
-  const bool is_head_rank = rank_v0 == communicator.rank();
+namespace {
```

Comment:
Is this intentional? Does it serve a purpose?

Reply:
No, it is just a leftover from when I was splitting the methods. It was a way to distinguish methods that were not directly used from the algorithm (opposed to `local::` and `distributed::`), but actually it does not serve any purpose, since the anonymous namespace will not play any role in hiding methods because they are called from within the same file. I'm going to remove it.